packages/eslint-config/shared.js

```diff
@@ -154,7 +154,8 @@ module.exports = {
         // 'recommended' rules we alter
         '@typescript-eslint/no-explicit-any': 'warn',
         '@typescript-eslint/no-var-requires': 'off',
-        '@typescript-eslint/ban-types': 'warn',
+        '@typescript-eslint/no-require-imports': 'off',
+        '@typescript-eslint/no-empty-object-type': 'off',
```

Comment:
I have turned this off just for the moment. It will be turned back on once I follow up with a PR to update the framework side config.